JobRunner/JobRunner.py

```diff
@@ -334,6 +334,16 @@ def run(self):
             self.logger.error("Failed to config . Exiting.")
             raise e
 
+        if "USE_SHIFTER" in os.environ:
```

Comment:
```suggestion
        if os.getenv("USE_SHIFTER"):
```
Do we need to verify that the environment variable is not empty or undefined?

Reply:
I think is kind of a hack, we could add a new variable like "RUNNING_AT_NERSC" or something, as we don't really care about shifter, we care about if its at nersc.

Reply:
👍